examples/the-main-function/the_main_function.out

```diff
@@ -2,30 +2,31 @@
 # `.bal` file, and execute the `bal run` command below.
 
 # Use the ballerina `run` command to invoke the `main` function specifying `Alice`
-# as the string argument for `name`. `18` would be set as the value for
-# `age` and `Freshman` would be set as the value for `year`.
-bal run the_main_function.bal Alice
+# as the command line string argument for `name`. `18` would be set as the value 
+# for `age` and `Freshman` would be set as the value for `year`.
+bal run the_main_function.bal -- --name=Alice
```

Comment:
IMO, it is better to stick to positional args (vs named args) for this feature.

If it is only the rest param that is not working atm, IMO we can remove just that bit from the example and keep the rest as is. And once the rest param issue is fixed we can add back the content.

Reply:
We changed it to use options for now. We plan to write a proper example when the PR fixing operand processing is merged for release. There has been a complete rewrite of the command line argument processing logic: https://docs.google.com/document/d/1Zcxe9jfS6pPsw5zISJ5HSZ6B1OxGK9cfbanjAd9FQcA/edit

Reply:
Oh okay, didn't know of this. But do these make sense as options? If not, should we consider removing the BBE until we can fully support this?

Reply:
@riyafa can we update it with new changes ?